map/src/drawer/components/tracks/LocalTrackGroup.jsx

```diff
@@ -0,0 +1,192 @@
+import {Button, Collapse, ListItemIcon, ListItemText, MenuItem, Typography} from "@mui/material";
+import {ExpandLess, ExpandMore, Folder} from "@mui/icons-material";
+import React, {useContext, useEffect, useState} from "react";
+import AppContext from "../../../context/AppContext";
+import TrackItem from "./TrackItem";
+import Utils from "../../../util/Utils";
+import {makeStyles} from "@material-ui/core/styles";
+import Actions from "./Actions";
+import CreatedTrackItem from "./CreatedTrackItem";
+import {styled} from "@mui/material/styles";
+import LocalTrackUtils from "../../util/LocalTrackUtils";
+import CreatedTrackUtils from "../../util/CreatedTrackUtils";
+
+const useStyles = makeStyles({
+    button: {
+        maxWidth: '120px !important',
+        maxHeight: '40px',
+        minWidth: '120px !important',
+        minHeight: '40px'
+    },
+    group: {
+        '& li': {
```

Comment:
Looks very similar to global style for other items if we need for all of them it's better to have it common

Reply:
This class used only for MenuItem. We want lines will be same height. We don't have do it other way.